include/cucumber-cpp/internal/step/StepManager.hpp

```diff
@@ -62,6 +62,11 @@ class InvokeArgs {
 
     template<class T> T getInvokeArg(size_type i) const;
     const Table & getTableArg() const;
+
+    size_type argsSize() const {
+      return args.size();
+    }
+
```

Comment:
Hi, sorry to chime in here.  I've used this extension in other code, not pushed or published anywhere (yet).  I needed it to pass it on to another code layer (if I recall correctly).